browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -1052,6 +1052,10 @@ test.describe('program creation', () => {
       await adminPrograms.expectFormFieldDisabled(FormField.PROGRAM_ELIGIBILITY)
       await adminPrograms.expectFormFieldDisabled(FormField.PROGRAM_CATEGORIES)
       await adminPrograms.expectFormFieldDisabled(FormField.APPLICATION_STEPS)
+      // Program external link is always enabled on pre North Star
+      await adminPrograms.expectFormFieldEnabled(
+        FormField.PROGRAM_EXTERNAL_LINK,
```

Comment:
praise: thank you for relying on element checks instead of adding new screenshots!